static/app/components/events/interfaces/threads/threadSelector/index.tsx

```diff
@@ -140,10 +244,23 @@ const ActiveThreadName = styled('span')`
 `;
 
 const StyledGrid = styled(ThreadSelectorGrid)`
-  padding-left: 40px;
-  padding-right: 40px;
+  padding-left: 36px;
+  padding-right: 20px;
   color: ${p => p.theme.subText};
   font-weight: ${p => p.theme.fontWeightBold};
   border-bottom: 1px solid ${p => p.theme.border};
   margin-bottom: 2px;
 `;
+
+const SortableThreadSelectorGridCell = styled(ThreadSelectorGridCell)`
+  cursor: pointer;
+  user-select: none;
+  border-radius: ${p => p.theme.borderRadius};
+  &:hover {
+    background-color: ${p => p.theme.backgroundSecondary};
+  }
+`;
+
+const HeaderText = styled(Flex)`
+  padding: 0 ${space(0.5)};
+`;
```

Comment:
nit - try not to style Flex since that has to insert 2 styles instead of one and makes me sad